### PR TITLE
[wrapper] style(*): remove "!important" usage

### PR DESCRIPTION
**Pull request by Barackos**

### CODE REVIEW

File: packages/editor/web/src/RichContentEditor/Toolbars/StaticToolbar/createFooterToolbar.jsx [hunk 1]
```diff
@@ -10,6 +10,7 @@ const getFooterTheme = theme => {
   return {
     toolbarStyles: {
       toolbar: classNames(toolbarStyles.footerToolbar, toolbarTheme && toolbarTheme.footerToolbar),
+      staticToolbar: classNames(theme.staticToolbar),
```
Comment (lxgreen) [inline packages/editor/web/src/RichContentEditor/Toolbars/StaticToolbar/createFooterToolbar.jsx:13]:
```suggestion
      staticToolbar: theme.staticToolbar,
```

File: packages/editor-common/web/src/Components/Dropdown.jsx [hunk 2]
```diff
@@ -203,9 +203,8 @@ class Dropdown extends Component {
       </div>
     ) : null;
 
-    const dropdownClass = classNames({
-      [styles['Dropdown-root']]: true,
-      [styles['Dropdown-root-isOpen']]: this.state.isOpen,
+    const dropdownClass = classNames(this.styles['Dropdown-root'], {
```
Comment (lxgreen) [inline packages/editor-common/web/src/Components/Dropdown.jsx:206]: c'mon this is not related to the task ;)

Comment (Barackos) [reply packages/editor-common/web/src/Components/Dropdown.jsx:206]: Oops XD
